src/Param.h

```diff
@@ -120,13 +120,19 @@ class Param {
     }
 
     void set_min_value(Expr min) {
+        if (!min.defined()) {
```

Comment:
This fix isn't really right -- it should be legal to call set_min_value(Expr()) to remove a previously-set constraint. We should figure out why it's segfaulting in AOT. Is there a simple testcase to repeat?